src/widgets/gauge.rs

```diff
@@ -460,19 +461,22 @@ mod tests {
     #[test]
     #[should_panic = "Percentage should be between 0 and 100 inclusively"]
     fn gauge_invalid_percentage() {
-        let _ = Gauge::default().percent(110);
+        let gauge = Gauge::default().percent(110);
+        dbg!(gauge);
```

Comment:
I'd prefer the `let _` approach on these (generally) for reasons previously discussed. Unless I'm missing a rationale for doing this?